cleanlab/filter.py

```diff
@@ -694,9 +697,9 @@ def _reduce_prune_counts(prune_count_matrix: np.ndarray, frac_noise: float = 1.0
 
 
 def find_predicted_neq_given(
-    labels: np.ndarray, pred_probs: np.ndarray, *, multi_label: bool = False
+    labels: LabelLike, pred_probs: np.ndarray, *, multi_label: bool = False
```

Comment:
Change docstring to "np.ndarray or list"

Reply:
good catch, done